src/index.ts

```diff
@@ -401,13 +407,16 @@ export class MetadataComponent extends BaseComponent {
         const $metadataGroup: JQuery = this._buildMetadataGroup(metadataGroup);
 				this._$metadataGroups.append($metadataGroup);
 				const $value: any = $metadataGroup.find(".value");
+				const $items: any = $metadataGroup.find(".item");
```

Comment:
Just to satisfy my own curiosity: why are we changing from value to item here? Is this a v2 vs. v3 thing that Manifesto irons out for us?

Reply:
The function previously just iterated over the values of the items (i.e. metadata text, not label), so we were missing the labels necessary to make the aria labels specific. I've adjusted the function to iterate over the items, and grab the item value and item label as needed.